### PR TITLE
Add Table of Contents to README

### LINKED ISSUE #547: Add Table of Contents to README
### Is your feature request related to a problem? Please describe.

The README has gotten rather long. It would be nice to have a table of contents so that I can quickly navigate to sections of interest.

### Describe the solution you'd like

Add a table of contents to `README.md`.

### Is this a breaking change

 * No, not breaking.

### PR DESCRIPTION
**Pull request by johnclary**
Fixes #547. 

I did make two changes to the README itself:

- added a header for **Endpoints** sections
- rename "Learn More / Get Involved / Contributing" to "Get Involved"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,33 +120,39 @@ More details and examples can be seen on our [Wiki](https://github.com/openmobil
...
 
-* [`django-mds`](https://github.com/polyconseil/django-mds) - Python/Django open source server for the [`agency`][agency] API, developed by BlueSystems.
+- [`django-mds`](https://github.com/polyconseil/django-mds) - Python/Django open source server for the [`agency`][agency] API, developed by BlueSystems.
 
 Please open a pull request if you create open source or private tools for implementing or working with MDS data.
 
 [agency]: /agency/README.md
 [provider]: /provider/README.md
 [policy]: /policy/README.md
+x
```
Comment (Retzoh) [inline README.md:158]: What is this `x` for, just a typo ?

**Comment by schnuerle** [comment]
Thanks John!  TOC is a great idea.  I think I'll incorporate this and other change into the 2 PRs I'm working on for the release, not just #546 but also the larger #544.

File: README.md (hunk 1 above)
Comment (schnuerle) [reply README.md:158]: Fixed typo in my merger.

**Comment by schnuerle** [comment]
I incorporated this into the [release-1.0.0 branch main readme](https://github.com/openmobilityfoundation/mobility-data-specification/blob/release-1.0.0/README.md).  Thank you John for the work!

Note I didn't change all the *'s to -'s.  I actually do prefer the `-` since it's one keystroke to type and it's what Github uses in their markdown example.  But `*` is used through most of the other MDS pages, and changing them all flags every line as a delete and addition, which makes it harder to see what's really been changed.  So I think we should leave them as is for these reasons.

**Comment by johnclary** [comment]
thanks @schnuerle and sounds good.

**Comment by johnclary** [comment]
thanks @schnuerle. ditto what you said about markdown bullets. this was actually the first time I used a [markdown linter](https://code.visualstudio.com/api/extension-guides/markdown-extension).

**Pull request closed without merging**